stack.yaml

```diff
@@ -1,16 +1,28 @@
 resolver: lts-13.26
+# Resolves hjsmin requiring language-javascript ==0.6, while the purescript
+# compiler requires language-javascript ==0.7.
+allow-newer: true 
 packages:
 - '.'
 extra-deps:
 - bytestring-trie-0.2.5.0
+- cborg-0.2.2.0
 - connection-0.3.1
 - happy-1.19.9
-- language-javascript-0.6.0.13
+- language-javascript-0.7.0.0
 - network-3.0.1.1
-- purescript-0.13.3
+- ../purescript
+- ../purescript/lib/purescript-ast
+- ../purescript/lib/purescript-cst
```

Comment:
I tried to point this to GitHub following the [yaml configuration section of the Stack docs](https://docs.haskellstack.org/en/stable/yaml_configuration), but ultimately the build fails with `purescript` considered a missing dependency. I'm not sure how to point these dependencies somewhere other than local paths for testing's sake.

```yml
# purescript 0.14.0-rc4
- github: purescript/purescript
  commit: 8663d00a27c2c36e229f3d2f98358e660c297335
  subdirs:
    - lib/purescript-ast
    - lib/purescript-cst
```

Reply:
CI won't build until this is resolved, but I've been able to test the 4 things we wanted to test locally and everything checks out.

Reply:
What happens if you add a “.” entry in the subdirs list?